packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -701,12 +1099,12 @@ class NavigatorObserver {
 /// ));
 /// ```
 ///
-/// The page route is built in two parts, the "page" and the
-/// "transitions". The page becomes a descendant of the child passed to
-/// the `transitionsBuilder` function. Typically the page is only built once,
-/// because it doesn't depend on its animation parameters (elided with `_`
-/// and `__` in this example). The transition is built on every frame
-/// for its duration.
+/// The route is built in two parts, the "page widget" and the "transitions". The
```

Comment:
Page isn't a widget, is it?

Reply:
we should rewrite this without using the word "page". Maybe "body".

Reply:
i guess since we call it a pageBuilder that would be hard. hrm.

Maybe we can deprecated pageBuilder and rename it bodyBuilder and have pageBuilder just forward to bodyBuilder (assert only one is non-null)

Reply:
We have lots of things that use the word Page, but they are not related to a Page in the Navigator 2.0 sense, e.g. PageRouteBuilder, MaterialPageRoute, PageRoute, etc. All of this is as confusing as the pageBuilder property here. Re-naming all of these seems like a pretty big API churn. Maybe we should instead find a different word for page in the new API and leave the existing things as-is?

Reply:
What term can we use instead?

Reply:
I don't have a great idea. Maybe we just leave things how they are for now.

Reply:
I don't have anything concrete neither, It seems like the right thing to do is to migrate all other classes that has name 'Page' in it because they are named after page for no reason and now we have a thing that can only be named 'Page' and nothing else. I agree we should leave it as is and migrate them if developers start to complain about the name is confusing with other classes.